storage/tests/unit/test_bucket.py

```diff
@@ -308,6 +308,25 @@ def test_create_w_explicit_project(self):
         self.assertEqual(kw['query_params'], {'project': OTHER_PROJECT})
         self.assertEqual(kw['data'], DATA)
 
+    def test_create_w_explicit_location(self):
+        PROJECT = 'PROJECT'
+        BUCKET_NAME = 'bucket-name'
+        LOCATION = 'continent-northeast1'
```

Comment:
use a possible location here (I understand it's not making a real request):
"us-central1"

Reply:
this is funny because I did and thought the exact opposite. I will make it real.